packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -31,10 +31,13 @@ import 'theme.dart';
 // late String _logoAsset;
 // double _myToolbarHeight = 250.0;
 
-typedef _FlexibleConfigBuilder = _ScrollUnderFlexibleConfig Function(BuildContext);
+typedef _FlexibleConfigBuilder = _ScrollUnderFlexibleConfig Function(
```

Comment:
please undo all the changes to formatting in this file, https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#formatting